tests/backends/stream/test_base.py

```diff
@@ -0,0 +1,19 @@
+"""Tests for Ralph base stream backend"""
+
+from ralph.backends.stream.base import BaseStream
+
+
+def test_backends_stream_base_abstract_interface_with_implemented_abstract_method():
+    """Tests the interface mechanism with properly implemented abstract methods."""
+
+    class GoodStream(BaseStream):
+        """Correct implementation with required abstract methods."""
+
+        name = "good"
+
+        def stream(self):
+            """Fakes the stream method."""
+
+    GoodStream()
```

Comment:
Is this to test that we don't get exceptions during instantiation ?
```suggestion
    try:
        GoodStream()
    except Exception:
        pytest.fail("A correct implementation of BaseStream should not raise exceptions")
```

Reply:
Yup. IMO it's overkill as it will fail with an explicit message mentioning that the abstract method should be implemented.